crml/governance/src/lib.rs

```diff
@@ -103,6 +118,8 @@ decl_storage! {
 		NextProposalId get(fn next_proposal_id): ProposalId;
 		/// Proposal bond amount in 'wei'
 		ProposalBond get(fn proposal_bond): Balance;
+		/// Minimum stake required to create a new council member
+		MinimumCouncilStake get(fn minimum_council_stake): Balance = 20_000;
```

Comment:
got to add the 0s to make it 4dp but lets leave this value as  `10_000_0000` to match the minimum stake default